package-lock.json

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
@iamikenna You can remove this file from this PR. 

You can also you .gitignore for ignoring this file while making a commit. 
https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files

Reply:
@iamikenna this should get fixed in this PR https://github.com/helpinghands2/helping_hands/pull/7. 
You can ignore the comment.